src/config.rs

```diff
@@ -106,40 +106,50 @@ pub struct ConfigCliPrompt {
 #[derive(Clone, Default, Debug, Deserialize)]
 pub struct RepodataConfig {
     /// Disable JLAP compression for repodata.
+    #[serde(alias = "disable-jlap")]
```

Comment:
```suggestion
    // BREAK: make the alias rename
    #[serde(alias = "disable-jlap")]
```
This is probably what you want to do for all these aliases :cry:

Reply:
For sure!